src/components/organisms/WithInformationCard/WithInformationCard.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import { AnyVenue } from "types/venues";
+
+import { InformationLeftColumn } from "components/organisms/InformationLeftColumn";
+import InformationCard from "components/molecules/InformationCard";
+
+import "./WithInformationCard.scss";
+
+interface PropsType {
+  venue: AnyVenue;
+  showInfo?: boolean;
+}
+
+export const WithInformationCard: React.FunctionComponent<PropsType> = ({
+  venue,
+  showInfo = false,
+}) => (
+  <>
+    {showInfo && (
+      <InformationLeftColumn iconNameOrPath={venue?.host?.icon || "info"}>
+        <InformationCard>
+          <p className="information-title">{venue.name}</p>
+          {venue.config?.landingPageConfig.subtitle && (
```

Comment:
Instead of repeating these long variable paths here directly in the JSX, it would be better to extract them into `const`s in the main body of this component above.

Eg. something like this:

```typescript
const { subtitle, description } = venue.config?.landingPageConfig;
```